### DISCUSSION TITLE
MacOS App: "App lock" Missing Features (and an extra requested option)

### DISCUSSION THREAD

**Discussion by limes-info** [ente-io/ente#3904] [Enhancements]
I've been using the iPadOS version on my Mac (forgot why), so today decided to switch it to the MacOS native version (since "Auto lock" wasn't working on the iPadOS version on MacOS).

The MacOS native version is missing the "Auto lock" option (setting to lock after certain app idle time).

Additionally I'd like to request an "On System Lock" option in addition to the time based options. (so when you lock your computer the app locks as well)

**Comment by NeverRIEght** [comment]
To be merged with #3929 ?

**Reply by limes-info** [reply]
Same feature request, different OS', so if the codebase is shared, yep merge.

If merged though:
I'd still like to request an "On System Lock" option in addition to the time based options, so when you lock your computer, the app locks as well (BitWarden does this)
